Documentation/Example.md

```diff
@@ -8,5 +8,5 @@ In order to open the example project you have to run in your terminal the follow
 
 ```bash
 cd Example
-xcodegen && xed .
+xcodegen && pod install && xed SBTUITestTunnel.xcworkspace
```

Comment:
`pod install` should not be needed since it is executed in the postGenCommand [here](https://github.com/Subito-it/SBTUITestTunnel/blob/master/Example/project.yml#L6). Also `xed .` should open the project/workspace it in the current working directory without requiring the full name to be specified."

Reply:
Hmm it wasn't working when I first started using the example project, but it is now.  I wonder if it was related to RVM on my local.  I'll revert this